src/main/java/com/example/money_diary_service/MoneyController.java

```diff
@@ -0,0 +1,22 @@
+package com.example.money_diary_service;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class MoneyController {
+
+    public MoneyController(Money_Diary_Mapper money_diary_Mapper) {
```

Comment:
型や変数名にアンダースコアは使わないですね。
```suggestion
    public MoneyController(MoneyDiaryMapper moneyDiaryMapper) {
```

Reply:
訂正しました。